package.json

```diff
@@ -31,9 +32,9 @@
 		"cli-dashboard:prebuild": "npm run build:remove -w @google-psat/cli-dashboard",
 		"cli-dashboard:build": "npm run cli-dashboard:prebuild && cross-env NODE_ENV=production npm run build -w @google-psat/cli-dashboard",
 		"cookie-db:update": "node scripts/update-cookie-db.cjs",
-		"ext:dev": "npm run dev -w @google-psat/extension",
+		"ext:dev": "npm run build:tsc-packages && npm run dev -w @google-psat/extension",
```

Comment:
`build:tsc-packages` runs all the `:build` instead of `:dev`. This means it will not pick up new changes made to other packages like library-detection while running `npm run ext:dev`

Reply:
Fixed in latest commits.